spring-kafka/src/main/java/org/springframework/kafka/aot/KafkaRuntimeHints.java

```diff
@@ -144,7 +144,7 @@ public void registerHints(RuntimeHints hints, @Nullable ClassLoader classLoader)
 					RangeAssignor.class,
 					RoundRobinAssignor.class,
 					StickyAssignor.class,
-					// standard partitioners
+					// standard partitions
```

Comment:
This is wrong. 
The next classes in this list are really `Partitioner` implementations.

Reply:
yeah, you are totally right. Correction reverted back. thx